optaplanner-wb-webapp-war/.gitignore

```diff
@@ -0,0 +1,16 @@
+/target
+/local
+/repositories
+/src/main/gwt-unitCache/
+/src/main/webapp/WEB-INF/classes/
```

Comment:
war generated files should go into target/....war/WEB-INF/classes ?

Reply:
Good question. Currently the ignore is justified because build output directory is set to `src/main/webapp/WEB-INF/classes`. Don't know if that's necessary. I'll see if we can get rid of that later.